lib/rules/stylelint-disable-no-empty/index.js

```diff
@@ -0,0 +1,54 @@
+"use strict"
+
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+
+const ruleName = "stylelint-disable-no-empty"
+
+const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected empty `stylelint-disable` comment",
+})
+
+const stylelintDisableCommand = "stylelint-disable"
+
+const rule = (actual) => {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) {
+      return
+    }
+
+    root.walkComments((comment) => {
+      if (!isDisableCommand(comment.text)) {
+        return
+      }
+
+      const specificRules = comment.text.split(" ", 2).slice(1)
```

Comment:
Tricky, indeed. I think this validation might actually work better not as a rule, but as a different kind of violation, like `reportNeedlessDisables`. I think:

- We should always report the disabling of a rule that does not actually exist.
- We could optionally report disables without a rule, `reportUnscopedDisables` or something.

You can see that [where `assignDisabledRanges`](https://github.com/stylelint/stylelint/blob/master/lib/lintSource.js#L89) is invoked, it has access to the configuration — so this is certainly possible.